libraries/botbuilder-ai-orchestrator/src/orchestratorRecognizer.ts

```diff
@@ -187,42 +189,33 @@ export class OrchestratorRecognizer extends AdaptiveRecognizer implements Orches
             entities: {},
         };
 
-        if (text === '') {
+        if (!text || text === '') {
             // nothing to recognize, return empty result.
             return recognizerResult;
         }
 
-        this._initializeModel();
-
-        if (this.externalEntityRecognizer) {
-            // Run external recognition
-            const externalResults = await this.externalEntityRecognizer.recognize(
-                dc,
-                activity,
-                telemetryProperties,
-                telemetryMetrics
-            );
-            recognizerResult.entities = externalResults.entities;
-        }
-
         // Score with orchestrator
-        const results = await this._resolver.score(text);
+        const results = await this._resolver?.score(text);
 
-        // Add full recognition result as a 'result' property
-        recognizerResult[this.resultProperty] = results;
+        if (results && results.length) {
```

Comment:
You want to add to recognizerResult[this.resultProperty] regardless if it's empty, this is for telemetry

Reply:
It's in there, it's just been moved within the if (results...) statement.